src/test/java/com/endava/service/IncomeExpenseServiceImplTest.java

```diff
@@ -267,14 +262,14 @@ public void testViewIncomeByIncorrectWherefrom() {
     public void testViewIncomeByCorrectWherefrom() {
         Wherefrom wherefrom = new Wherefrom();
         wherefrom.setName("salary");
-        List<Income> incomes = new ArrayList<>( Arrays.asList(income1, income2, income3, income4) );
+        List<MoneyTransfer> moneyTransfers = new ArrayList<>( Arrays.asList(moneyTransfer, moneyTransfer1, moneyTransfer2, moneyTransfer3) );
 
-        when(incomeDao.readAll()).thenReturn(incomes);
+        when(moneyTransferDao.readAll()).thenReturn(moneyTransfers);
 
-        List<Income> actualIncomes = incomeExpenseService.viewIncomesByWherefrom(wherefrom);
-        List<Income> expectedIncomes = new ArrayList<>( Arrays.asList(income3, income4) );
+        List<MoneyTransfer> actualIncomes = incomeExpenseService.getMoneyTransferByWherefrom(wherefrom);
+        List<MoneyTransfer> expectedIncomes = new ArrayList<>( Arrays.asList(moneyTransfer2, moneyTransfer3) );
```

Comment:
You don't need new ArrayList here. Arrays.asList returns unmodifiable List, right what you need :)